cli/flox-rust-sdk/src/providers/build.rs

```diff
@@ -140,24 +164,30 @@ impl ManifestBuilder for FloxBuildMk {
         packages: &[String],
     ) -> Result<BuildOutput, ManifestBuilderError> {
         let mut command = self.base_command(flox, base_dir);
+        command.arg("build");
+        command.arg(format!("BUILDTIME_NIXPKGS_URL={}", &*BUILDTIME_NIXPKGS_URL));
         command.arg(format!("FLOX_ENV={}", built_environments.develop.display()));
         command.arg(format!(
             "FLOX_ENV_OUTPUTS={}",
             serde_json::json!(built_environments)
         ));
         command.arg(format!("FLOX_INTERPRETER={}", flox_interpreter.display()));
 
-        // Add build target arguments by prefixing the package names with "build/".
-        // If no packages are specified, build all packages.
-        // While the default target is "build", we explicitly specify it here
-        // to avoid unintentional changes in behavior.
-        if packages.is_empty() {
-            let build_all_target = "build";
-            command.arg(build_all_target);
-        } else {
-            let build_targets = packages.iter().map(|p| format!("build/{p}"));
-            command.args(build_targets);
-        };
+        // Add the list of packages to be built by passing a space-delimited list
+        // of pnames in the PACKAGES variable. If no packages are specified then
+        // the makefile will build all packages by default.
+        command.arg(format!("PACKAGES={}", packages.join(" ")));
```

Comment:
Are there any possible quoting/word-splitting issues on the `make` side by using spaces instead of commas here?

Reply:
No, spaces are the natural word delimiter for make and rust is ensuring that the whole string is passed in a single argv variable so a multi-word PACKAGES value is passed correctly.

Reply:
FYI, we _do_ allow packages named `"foo with spaces"` (well basically anything that parses as a toml key). We may want to restrict that eventually on the parsing i guess, but for now this will likely get you strange errors.